api/health.go

```diff
@@ -0,0 +1,42 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/gotify/server/model"
+)
+
+// The HealthDatabase interface for encapsulating database access.
+type HealthDatabase interface {
+	Ping() error
+}
+
+// The HealthAPI provides handlers for the health information.
+type HealthAPI struct {
+	DB HealthDatabase
+}
+
+// Health returns health information.
+// swagger:operation GET /health health getHealth
+//
+// Get health information.
+//
+// ---
+// produces: [application/json]
+// responses:
+//   200:
+//     description: Ok
+//     schema:
+//         $ref: "#/definitions/Health"
+func (a *HealthAPI) Health(ctx *gin.Context) {
+	if err := a.DB.Ping(); err != nil {
+		ctx.JSON(200, model.Health{
```

Comment:
Maybe `503 Service Unavailable` for this?

Reply:
I'll go with 500, because then it is consistent with the other apis when the db is not available